src/ansys_sphinx_theme/cheatsheet.py

```diff
@@ -27,7 +27,7 @@
 """
 
 import pathlib
-import subprocess
+import subprocess  # nosec: B404
```

Comment:
All nosec statements should be followed with at least an in-line comment justifying this decision right above

Reply:
Sure, added in https://github.com/ansys/ansys-sphinx-theme/pull/754/commits/8b7f66207866ed5f3ce1e33dccbd67eab077e0ca